app/Components/Sudoku Board/SudokuBoard.tsx

```diff
@@ -115,48 +119,65 @@ const SudokuBoard = (props: SudokuBoardProps) => {
    * @param inputValue User input 0-9
    */
   const updateCellEntry = (inputValue: number) => {
-    if (sudokuBoard.selectedCell == null) {
-      return;
-    }
-    const currentSelectedCell = getCurrentSelectedCell() as CellProps;
-    // We do not need to take action if this is a given value
-    if (currentSelectedCell.type === "given") {
+    if (sudokuBoard.selectedCell.length === 0) {
       return;
     }
 
-    const r: number = sudokuBoard.selectedCell.r;
-    const c: number = sudokuBoard.selectedCell.c;
-    const currentEntry = currentSelectedCell.entry;
-    const currentType = currentSelectedCell.type;
+    const newActionHistory: GameAction[] = [];
+    let cellsHaveUpdates = false;
 
-    // We do not need to take action if current value matches existing value, or if value is correct
-    if (
-      currentType === "value" &&
-      (currentEntry === inputValue ||
-        isValueCorrect(
-          sudokuBoard.puzzleSolution[r][c],
-          currentEntry as number
-        ))
-    ) {
+    const currentSelectedCells = getSelectedCells();
+
+    // We do not take action if more than one cell is selected and we are not in note mode
```

Comment:
would like to be able to delete notes from multiple cells at once even if not in note mode

Reply:
This has been fixed!